tests/utils/test_downstream.py

```diff
@@ -0,0 +1,134 @@
+        "minimal_golden_records_golden_records_draft",
+        "minimal_mastering_unified_dataset_dedup_high_impact_questions",
+        "minimal_mastering_unified_dataset_dedup_test_records_accuracy",
+        "minimal_mastering_unified_dataset_dedup_model",
+        "minimal_golden_records_source_list",
+        "minimal_mastering_unified_dataset_dedup_all_persistent_ids",
+        "minimal_mastering_unified_dataset_dedup_human_signals",
+        "minimal_mastering_unified_dataset_dedup_cluster_similarities",
+        "minimal_mastering_unified_dataset_dedup_non_null_count",
+        "minimal_mastering_unified_dataset_dedup_dnf_signals",
+        "minimal_mastering_unified_dataset_dedup_published_cluster_counts",
+        "minimal_golden_records_golden_records_pinned_cluster_input",
+        "minimal_mastering_unified_dataset_dedup_cluster_stats",
+        "minimal_mastering_unified_dataset_dedup_signals_predictions",
+        "minimal_golden_records_golden_records_rule_output",
+        "minimal_mastering_unified_dataset_dedup_clusters_with_data",
+        "minimal_mastering_unified_dataset_dedup_signals",
+    }
+    assert PIPELINE_DEPENDENT_DATASETS.issubset(set([d.name for d in downstream_datasets]))
+
+
+@mock_api(enforce_online_test=ONLINE_TEST)
+def test_get_dataset_downstream_dependencies_suggest_name():
+    client = tamr_toolbox.utils.client.create(**CONFIG["toolbox_test_instance"])
+    dataset = client.datasets.by_resource_id(MASTERING_UNIFIED_DATASET_ID)
+    downstream_datasets = tamr_toolbox.utils.downstream.datasets(
+        dataset, include_dependencies_by_name=True
+    )
+    # check returned results
+    # in newer version of Tamr, this number might increase
+    # because more persistence data is moved to HBase
+    assert len(downstream_datasets) >= 49
+    DEPENDENT_DATASETS_BY_NAME = {
+        "minimal_mastering_unified_dataset_dedup_verified_cluster_members",
+        "minimal_mastering_unified_dataset.recordPairLabel",
+        "minimal_mastering_unified_dataset.record_pair_feedback_uid",
+        "minimal_mastering_unified_dataset.record_pair_id_string",
+        "minimal_mastering_unified_dataset.internal_links",
+        "minimal_mastering_unified_dataset_dedup_suggested_clusters_log",
+        "minimal_mastering_unified_dataset.cluster_feedback",
+        "minimal_mastering_unified_dataset.internal_comments",
+        "minimal_mastering_unified_dataset.global_record_id",
+    }
+    assert DEPENDENT_DATASETS_BY_NAME.issubset(set([d.name for d in downstream_datasets]))
+
+
+@mock_api(enforce_online_test=ONLINE_TEST)
+def test_get_downstream_projects():
+    client = tamr_toolbox.utils.client.create(**CONFIG["toolbox_test_instance"])
+    dataset = client.datasets.by_resource_id(MASTERING_UNIFIED_DATASET_ID)
+    downstream_projects = tamr_toolbox.utils.downstream.projects(
+        dataset, include_dependencies_by_name=True
+    )
+    assert len(downstream_projects) == 2
+    assert (
```

Comment:
Checking the returned project ids directly will be more robust. If for instance another parameter was added to the string representation of a Project.